source/common/common/base64.h

```diff
@@ -43,6 +43,15 @@ class Base64 {
    * bytes.
    */
   static std::string decode(const std::string& input);
+
+  /**
+   * Base64 decode an input string. Padding is not required.
+   * @param input supplies the input to decode.
+   *
+   * Note, decoded string may contain '\0' at any position, it should be treated as a sequence of
+   * bytes.
+   */
+  static std::string decode_without_padding(const std::string& input);
```

Comment:
Nit: `decodeWithPadding`

Reply:
Make this take absl::string_view as parameter, then you can save one copy in caller side. The content of the method should just work without modification.

Reply:
Done x 2. Nice!